docs/learn/sandboxes/index.md

```diff
@@ -51,56 +51,59 @@ Complete list of commands and their parameters can be found when running `prd --
 
 ## Available configuration options
 
-The only necessary configuration is in the `credentials.json` file right after running `prd init`.
-
-Default configuration using username and password:
+The only necessary configuration is in the `credentials.json` and ini the `prd_config.yaml` files right after running `prd init`.
+First, we will setup izr credentials to the `credentials.json` file, where we can work with `username` + `password` combination (Example 1), or there is alternative with the `token` in the Example 2.
 
+#### Example 1
+Authentication using username and password.
 ```json
 {
   // Source organization (typically the only one needed).
   "source": {
-    // API base URL. Consult with your support team (SA) in case you are not sure what
-    // the value should be.
-    "api_base": "https://api.elis.rossum.ai/v1",
-
     // Username under which `prd` will be calling the API.
     "username": "...",
-
```

Comment:
@VaclavRut I do not have a big issue with this, just a note that the empty lines were there on purpose for better readability. 😎